lib/suspenders/adapters/heroku.rb

```diff
@@ -37,6 +37,15 @@ def set_heroku_rails_secrets
         end
       end
 
+      def set_heroku_backup_schedule
+        %w(staging production).each do |environment|
+          run_toolbelt_command(
+            "pg:backups:schedule DATABASE_URL --at '02:00 America/Los_Angeles'",
```

Comment:
It's the Heroku default recommended in their docs -- to convert it do you mean 2AM UTC or 10AM UTC?